resources/resources.md

```diff
@@ -0,0 +1,16 @@
+---
+nav_order: 121
+has_children: true
+title: Resources
+description: Machine translation resources
+---
+
+### Calls for papers
+
+| Publication | Organisers | Deadline | Type |
```

Comment:
I think we should def have https://amtaweb.org/amta-2022-1st-call-for-papers-presentations-and-workshop-and-tutorial-proposals/.

We can also add WMT22 already, even if it's just "June 2022"